warehouse/models/mart/gtfs_quality/fct_monthly_reports_site_organization_guideline_checks.sql

```diff
@@ -42,13 +46,19 @@ fct_monthly_reports_site_organization_guideline_checks AS (
         idx.publish_date,
         checks.feature,
         checks.check,
-        checks.status
+        checks.reports_status,
+        CASE
+            WHEN guideline_manual_checks.check IS NOT null THEN 1
+            ELSE 0
+        END AS is_manual 
     FROM idx_monthly_reports_site AS idx
     LEFT JOIN generate_biweekly_dates AS dates
         USING (publish_date)
-    LEFT JOIN checks
+    JOIN checks
```

Comment:
Just curious as to the rationale for changing this from a left to an inner join? We want to drop rows that won't have checks? Either way would prefer to specifically say `INNER JOIN` even though it is redundant but I think we do that for consistency (and maybe SQLFluff yells about it somewhere?)

Reply:
Yes exactly that - there were many rows with null check values, which I ended up filtering out in python within reports site infra. I figured it was cleaner to filter further upstream - i will leave a comment about that and switch to INNER JOIN